ffi/capi/src/normalizer.rs

```diff
@@ -74,6 +74,21 @@ pub mod ffi {
             let s = s.as_bytes(); // #2520
             self.0.is_normalized_utf8(s)
         }
+
+        /// Return the index a slice of potentially-invalid UTF-8 is normalized up to
+        #[diplomat::rust_link(
```

Comment:
Not sure how `experimental` works with FFI, but committing to this FFI as non-experimental requires deciding on the exact semantics of `is_normalized_utf8_up_to` in the case of not-well-formed input.

Reply:
Generally we don't have experimental code in FFI

Reply:
Should I just remove it or comment it out as a todo?

Reply:
I think we're not considering this experimental?

Please make this use the `&DiplomatStr` type, which is an alias for `&[u8]`

Reply:
I think we should not consider this experimental. Then FFI exposure is OK.

Reply:
Unmarked as experimental.

Reply:
> Please make this use the `&DiplomatStr` type, which is an alias for `&[u8]`.

A bit confused, I can't find this used elsewhere. Also, do you mean eg `let s: &DiplomatStr` instead of the default inferred type?

Reply:
The other methods in this file have been migrated to `DiplomatStr` in https://github.com/unicode-org/icu4x/pull/4353, you need to sync

Reply:
Oh! My mistake, apologies.